src/containers/floating/XSpaceBanner/XSpaceBanner.tsx

```diff
@@ -15,6 +17,8 @@ import XSpaceSvg from '@app/components/svgs/XSpaceSvg';
 import { open } from '@tauri-apps/plugin-shell';
 import { XSpaceEventType } from '@app/utils/XSpaceEventType';
 import { useTranslation } from 'react-i18next';
+import { formatDateForEvent } from './formatDate';
+import { AnimatePresence } from 'framer-motion';
```

Comment:
```suggestion
import { AnimatePresence } from 'motion/react';
```

moved over to `motion` a little while back sorry 😬